.github/workflows/CI.yml

```diff
@@ -91,7 +91,7 @@ jobs:
         conda activate test
         if [[ "$RDKIT" == true ]]; then conda install rdkit; fi
         if [[ "$OPENEYE" == true ]]; then
-          conda install openeye-toolkits=2019
+          conda install openeye-toolkits
```

Comment:
I could see the case for `openeye-toolkikts >=2020` ... probably better to leave it off - how much do we worry about users having old versions of the toolkit installed? If they _did_ (and they shouldn't) then the errors may be cryptic.

Reply:
This should actually be reverse-compatible! So there's no need to pin OE or worry about older versions :-)